ui/src/components/Item/Preview.svelte

```diff
@@ -80,14 +80,14 @@
     if ($state.isLoaded && !item) {
       return api.portal.do.subscribe(key);
     }
-    if (groupsStrucs.includes(item.keyObj.struc)) isGroupsItem = true;
+    if (groupsStrucs.includes(item?.keyObj?.struc)) isGroupsItem = true;
```

Comment:
i'm guessing because now getGroupsFeed is manipulated outside of a reactive statement, these are necessary for data that hasn't come thru yet. unsure